cue/model/api/v1/secret/oauth_go_gen.cue

```diff
@@ -0,0 +1,22 @@
+// Code generated by cue get go. DO NOT EDIT.
+
+//cue:generate cue get go github.com/perses/perses/pkg/model/api/v1/secret
+
+package secret
+
+import (
+	"net/url"
+	"golang.org/x/oauth2"
+)
+
+#PublicOAuth: {
+	clientID:         #Hidden @go(ClientID)
+	clientSecret:     #Hidden @go(ClientSecret)
+	clientSecretFile: string  @go(ClientSecretFile)
+	tokenURL:         string  @go(TokenURL)
+	scopes: [...string] @go(Scopes,[]string)
+	endpointParams: url.#Values       @go(EndpointParams)
+	authStyle:      oauth2.#AuthStyle @go(AuthStyle)
```

Comment:
Cuelang package `golang.or/x/oauth2` does not exist and so the cuelang tests are failing. 

`@oauth2.#AuthStyle is a number.` @AntoineThebaud what should we do to fix this issue ?